src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="rating stars--0">
+      <div class="rating__item"></div>
+      <div class="rating__item"></div>
```

Comment:
`--0` це модифікатор блоку `starts` але такого блоку в тебе немає, модифікатор має прив'язуватись до вже існуючого блоку, можна зробити так
```suggestion
    <div class="starts stars--0">
      <div class="starts__item"></div>
      <div class="starts__item"></div>
```